flink-table-store-core/src/main/java/org/apache/flink/table/store/file/data/AppendOnlyCompactManager.java

```diff
@@ -97,31 +108,90 @@ LinkedList<DataFileMeta> getToCompact() {
         return toCompact;
     }
 
-    /** A {@link CompactTask} impl for append-only table. */
-    public static class AppendOnlyCompactTask extends CompactTask {
+    /** A {@link CompactTask} impl for ALTER TABLE COMPACT of append-only table. */
+    public static class RollingCompactTask extends CompactTask {
```

Comment:
Can we add more description comments?
It dose not look like a `Rolling`, maybe `IterativeCompactTask`?